assignment_3/Marksheet/src/Marksheet.java

```diff
@@ -0,0 +1,148 @@
+import java.util.Scanner;
+
+
+public class Marksheet
+{
+	/**
+	 * 
+	 * @param grades of students
+	 * @return Average of grades
+	 */
+	public float Calculate_Average(int grades[])
+	{
+		float Average=0;
+		int sum=0;
+		for(int i=0;i<grades.length;i++)
+		{
+			sum=sum+grades[i];
+		}
+		
+		Average=(float) (sum/grades.length);
```

Comment:
It will fail if grades.length = 0
Handle exceptions properly.
Provide null checks
always check if value is 0 while performing division